rbx_types/src/content.rs

```diff
@@ -55,6 +55,24 @@ impl Content {
     pub fn into_value(self) -> ContentType {
         self.0
     }
+
+    /// Returns Some(&str) if `Content` is a Uri.
```

Comment:
This isn't a very helpful doc comment. It's essentially just saying what the type signature already communicates. I'd prefer something like this:
```suggestion
    /// If this `Content` is a URI, returns the URI. Otherwise, returns `None`.
```